Makefile

```
@@ -0,0 +1,73 @@
+DBG ?= 0
+
+ifeq ($(DBG),1)
+GOGCFLAGS ?= -gcflags=all="-N -l"
+endif
+
+VERSION     ?= $(shell git describe --always --abbrev=7)
+REPO_PATH   ?= github.com/openshift/cluster-api-provider-equinix-metal
+LD_FLAGS    ?= -X $(REPO_PATH)/pkg/version.Raw=$(VERSION) -extldflags -static
+
+GO111MODULE = on
+export GO111MODULE
+GOFLAGS ?= -mod=vendor
+export GOFLAGS
+GOPROXY ?=
+export GOPROXY
+
+NO_DOCKER ?= 0
+ifeq ($(NO_DOCKER), 1)
+  DOCKER_CMD =
+  IMAGE_BUILD_CMD = imagebuilder
+  CGO_ENABLED = 1
+else
+  DOCKER_CMD := docker run --rm -e CGO_ENABLED=1 -v "$(PWD)":/go/src/github.com/openshift/cluster-api-provider-equinix-metal:Z -w /go/src/openshift/cluster-api-provider-equinix-metal openshift/origin-release:golang-1.15
```

looks like the workdir is off

```suggestion
  DOCKER_CMD := docker run --rm -e CGO_ENABLED=1 -v "$(PWD)":/go/src/github.com/openshift/cluster-api-provider-equinix-metal:Z -w /go/src/github.com/openshift/cluster-api-provider-equinix-metal openshift/origin-release:golang-1.15
```